tests/scripts/test-e2e-standalone.sh

```diff
@@ -15,6 +15,21 @@ set -eou pipefail
 # 3) Invoke the test-e2e Makefile target
 # 4) Terminate the OLS server
 
+function wait_for_ols() {
+  local timeout=120
+  local attempts=20
+  local sleep_for=$(( timeout / attempts ))
+  for i in {1..$attempts}; do
+    echo Checking OLS readiness, attempt "$i" of $attempts
+    curl -sk --fail "$1/readiness"
+    if [ $? -eq 0 ]; then
+      return 0
+    fi  
+    sleep $sleep_for
+  done
+  return 1
+}
```

Comment:
@syedriko i will replace this w/ the utils.sh import after the file movement is complete, right now the CI job is still invoking this script from another location so the import fails.

Reply:
your syntax for making the count variable is not working.  I am setting this back to hardcoding it and you can submit a PR improvin the syntax if you can make it work.

Reply:
specifically ` for i in {1..$attempts}; do` does not seem to be valid, i don't think you can reference a variable in that spot.

Reply:
Of course not
```
  for ((i=1; i<=$attempts; i++)); do
```
works well though.